tests/unit/components/test_dataset_storage.py

```diff
@@ -122,3 +129,5 @@ def test_categories_transform(
         actual = set(cat.name for cat in storage.categories()[AnnotationType.label])
         expect = set(mapping.values())
         assert actual == expect
+
```

Comment:
With `StreamDatasetStorage`, `storage.infos()` returns only `self.stacked_transform.infos()`. What happens if there is no transform in `StreamDatasetStorage`?

Reply:
It actually has a problem. Therefore, I made a bug ticket in the JIRA (https://jira.devtools.intel.com/browse/CVS-115725). https://github.com/openvinotoolkit/datumaro/pull/1101/commits/0f18b96cf74ce2506d6714db6f0a727cef37942e resolves it and it also adds the tests which will be failed if it is not corrected.

Reply:
This problem is not exactly what you ask. Therefore, the answer to your question is that just return the extractor's `infos()`. Please see this test:
https://github.com/openvinotoolkit/datumaro/blob/32317f1f40894cd3bf6ab8b95c84777eebb43a0c/tests/unit/components/test_dataset_storage.py#L62-L66